htdocs/langs/de_DE/products.lang

```diff
@@ -341,3 +341,4 @@ ErrorDestinationProductNotFound=Zielprodukt nicht gefunden
 ErrorProductCombinationNotFound=Produktvariante nicht gefunden
 ActionAvailableOnVariantProductOnly=Action only available on the variant of product
 ProductsPricePerCustomer=Product prices per customers
+ProductSupplierExtraFields=Ergänzende Attribute (Lieferantenpreise)
```

Comment:
New translation must be added in file en_US.
Adding en de_DE is useless, it is erased and removed by transifex sync if not added into en_US file.